src/BitcoinHelpers.js

```diff
@@ -105,7 +105,7 @@ const BitcoinHelpers = {
          * @param {string} pubKeyHash A pubKeyHash as a string.
          * @param {string} network The Bitcoin network for the Bech32 address.
          *
-         * @return {string} A Bech32 address to 
+         * @return {string} A Bech32 address to
          */
         pubKeyHashToBech32: function(pubKeyHash, network) {
```

Comment:
I don't think we're using this function any more, now that redemption uses the raw outputScript? [Quick sourcegrep](https://sourcegraph.com/search?q=repo:%5Egithub%5C.com/keep-network/tbtc%5C.js%24%40arc-reactor+pubKeyHashToBech32&patternType=literal) didn't turn up usages.

Reply:
This is true, wasn't ready to drop it yet though. Trying to think through the utility of it for a third-party API rather than just our own usage of it…

Reply:
Reflecting on design principles I'm learning - Piotr's mentioned to me in the past that it's good to keep the interface minimal. On the other hand, given that bcoin docs are somewhat scarce, I see it could be useful.

Reply:
Product and library have different approaches on the minimal interface front, however. In this case I’m not certain it deserves to survive —just wasn’t ready to kill it yet.